app/controllers/batches_controller.rb

```diff
@@ -0,0 +1,24 @@
+class BatchesController < ApplicationController
+    def index
+        if user_signed_in?
+            @batches = Batch.where(:user_id == current_user.id)
```

Comment:
You can also do ```current_user.batches```, which is a little more rubyish, as long as you have the has_many association setup correctly.